src/main/java/executor/command/CommandList.java

```diff
@@ -7,8 +7,14 @@
 public class CommandList extends Command {
 
     // Constructor
+    /**
+     * Constructor for CommandList subCommand Class.
+     * @param userInput The user input from the CLI
+     */
     public CommandList(String userInput) {
         this.userInput = userInput;
+        this.description = "Lists all the entries";
```

Comment:
Could you specify it to be lists all the tasks and reciepts added in by the user?